tests/FINDOLOGIC/Tests/Objects/XmlResponseTest.php

```diff
@@ -206,6 +206,27 @@ public function testResponseWillReturnItemsAsExpected()
             '', // Category
         ];
 
+        $response = $this->getRealResponseData();
```

Comment:
Hm.... At the moment I can not tell a Filter to take in an array of data, just a SimpleXmlElement is accepted. My idea was something like that:

```php
$actualResponse = [
    'price' => new Filter([
        'name' => 'price',
        'display' => 'Blubbergurken',
        . . .
    ]),
    'Farbe' => new Filter([
        'name' => 'Farbe',
        'display' => 'Blubbergurken2',
    ]),
];
```

Not sure if I should implement that.

Reply:
Yes okay I will do that but only after merging this into master. Because I have another PR open where I make those tests more strict and otherwise I will have massive merge conflicts.